appinventor/components/src/com/google/appinventor/components/runtime/ChartDataBase.java

```diff
@@ -32,6 +32,11 @@
     // where each index corresponds to a single dimension.
     protected List<String> dataFileColumns;
 
+    // Properties used in Designer to import from Web components.
+    // Represents the names of the columns to use,
+    // where each index corresponds to a single dimension.
+    protected List<String> webColumns;
```

Comment:
Implement Web Column logic in Designer (hide when non-Web component, hide/show Data Source value)